### ISSUE TITLE
add tabs on the nav menu

### ISSUE THREAD

**Issue by vioil** [vivoconference/vivoconference.github.io#69]
Please add REGISTER  ATTEND  SPEAKERS tabs on the top nav bar.
Link REGISTER to the Eventbrite page: https://www.eventbrite.com/e/vivo-2019-conference-tickets-50867496050

**Comment by hudajkhan** [comment]
The site has been updated with the register tab.  Would you also like the register button (see http://vivoconference.org/vivo2018/ for reference) under the main title?
 For the attend tab, we'd need to update the details on the attend page  (see http://vivoconference.org/vivo2018/attend/ for reference) .  If we added the speakers page (see http://vivoconference.org/vivo2018/speakers/ for reference), for now we would just have the invited speakers on that list.  
On a completely different note, would it be possible to get rid of the typeout for "Xth international conference" and just have regular text there?

**Comment by vioil** [comment]
Yes, definitely under the title we should have the REGISTER button.

Let's wait for ATTEND tab

Let's go and add the speakers we have so far - they will appear at the top anyway. 

Yes, we can get rid of the typeout.

**Comment by vioil** [comment]
Updating this ticket to add the Call for Proposal tab up on the nav bar. Link to the page here: 
http://vivoconference.org/talk-proposals/
@hudajkhan can you work on this please?
